test/presenters/licences/view-licence.presenter.test.js

```diff
@@ -1120,6 +1123,40 @@ describe('View Licence presenter', () => {
       })
     })
   })
+
+  describe("the 'roles' property", () => {
+    describe('when the authenticated user has roles', () => {
+      beforeEach(() => {
+        auth = {
+          credentials: {
+            roles: [{
+              role: 'role 1'
+            },
+            {
+              role: 'role 2'
+            }]
+          }
+        }
+      })
+
+      it('returns the roles in a flat array', () => {
+        const result = ViewLicencePresenter.go(licence, licenceAbstractionConditions, auth)
+
+        expect(result.roles).to.equal(['role 1', 'role 2'])
+      })
+    })
+    describe('when the authenticated user has NO roles', () => {
+      beforeEach(() => {
+        auth = undefined
+      })
+
+      it('returns the roles in a flat array', () => {
```

Comment:
Guessing a copy & paste typo

```suggestion
      it('returns null for the roles', () => {
```